PICS/Patient Information/HeightKey.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  HeightKey.swift
+//  PICS
+//
+//  Created by Catherine Zhang on 1/30/24.
```

Comment:
Possibly need to follow the header comments from other existing files and include the license info (e.g. SPDX-FileCopyrightText: 2023 Stanford University and SPDX-License-Identifier: MIT) to pass the REUSE Compliance Check to merge. Same for the other new file.